intellij-ext/src/main/kotlin/com/metalbear/mirrord/GoRunConfig.kt

```diff
@@ -0,0 +1,38 @@
+package com.metalbear.mirrord
+
+import com.goide.execution.GoRunConfigurationBase
+import com.goide.execution.GoRunningState
+import com.goide.execution.extension.GoRunConfigurationExtension
+import com.intellij.execution.configurations.RunnerSettings
+import com.intellij.execution.target.TargetedCommandLineBuilder
+
+class GoRunConfig : GoRunConfigurationExtension() {
```

Comment:
Why are these empty function implementations necessary?

Reply:
`GoRunConfigurationExtension` is an abstract class, those methods are to be implemented